1209/6809/index.md

```diff
@@ -3,8 +3,9 @@ layout: pid
 title: Doppelganger
 owner: TachRadio
 license: BSD-2-Clause
-site: https://thewired.doesntexist.org
+site: https://thewireddoesntexist.org
 source: https://github.com/NilujePerchut/doppelganger_hardware
-	https://github.com/NilujePerchut/doppelganger_soft
```

Comment:
Personally I think it's better to put the software source here, and the hardware in the body, as this is a website about PIDs, i.e. software, rather than hardware.